src/tools/tools/file.fs

```diff
@@ -309,7 +309,7 @@ module  FileTools =
                                 nVerified <- -1 
                             else
                                 Threading.Thread.Sleep( 5 )
-            Logger.LogF(LogLevel.MediumVerbose,  fun _ -> sprintf "WriteBytesToFileConcurrentPCompare: done verify file '%s' (verified = %d), detail: %s" filename nVerified message)
+            //Logger.LogF(LogLevel.MediumVerbose,  fun _ -> sprintf "WriteBytesToFileConcurrentPCompare: done verify file '%s' (verified = %d), detail: %s" filename nVerified message)
```

Comment:
why this line was commented out? It was added intentionally to help track down an issue.